### PR TITLE
[First dwaft] Dimensions.md

### PR DESCRIPTION
**Pull request by tralezab**
I really have had nothing to do these past few days, so I wanted to try tackling the explanation of the "bullshit science magic" that is bluespace magic and giving a little background on it and the other dimensions that show up while playing ss13. As per the instructions of shaps on contributing I've purposefully left it fairly vague and great for people to build on in neat discussions in game.

### CODE REVIEW

**Comment by Cyberboss** [comment]
Maybe mention subspace? Is that still a thing?

**Comment by Ryll-Ryll** [comment]
Interesting idea for an article, and the writing style seems fine! A few notes though:

1. I envision bluespace as the main form of FTL travel in the SS13 universe, and while it's still highly experimental and rapidly developing (especially on the Periphery), [it's established in the article for Nanotrasen that basic understandings of bluespace exist back in Sol](https://github.com/tgstation/common_core/blob/master/Nanotrasen.md). NT's use of bluespace for things like bags of holding, artillery batteries, and highly accurate ship engines are all leagues ahead of TerraGov's best labs (who are probably still perfecting sending an apple from one side of a continent to another), but I think moving the discovery back to 2520 or so would fit better.
2. Using the word teleportation implies that the movement is instantaneous, which while it practically would be for short range jumps (like up to a few miles), there'd still be a noticeable travel time when you scale things to the size of space ships and moving around the galaxy. Scientists describing things love technical disclaimers, so I think some mention of that would fit an article like this.
3. We gotta get a less minecrafty name for the demon dimension

**Comment by itseasytosee** [comment]
A while ago I had the idea of bluespace technology utilizing a 4-dimensional realm as a way to travel from place to place near instantaneously. Ever read flatland?

File: Dimensions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Reebe is a white dimension followers of Ratvar said was created by Ratvar themselves. It was being used as a pocket dimension for the cultists to hide from Nanotrasen and fulfill whatever evil plot they had.
+
+Knowledge about Reebe is incredibly limited given the only known way to reach it died along with the cultists, who would mysteriously travel freely to and from.
+
+Trivia: If the creation story of Reebe is to be believed, then you could infer that Nar'Sie would also have one. Is the dimension demons come from Nar'Sie's dimension?
+
+## Netherworld
+
+The Netherworld is one of the few inhabited dimensions, with the inside being a vast plane of fully lit rainbow structures, flora, and sky. The dimension is survivable atmospheric amd pressure wise, but is filled with esoteric creatures of all sizes and shapes and colors, ranging from the size of ants to large buildings. The natural fauna within seem to be interlocked in a constant hyper evolutionary race, making a documentation of species impossible.
```
Comment (Fikou) [inline Dimensions.md:23]:
```suggestion
The Netherworld is one of the few inhabited dimensions, with the inside being a vast plane of fully lit rainbow structures, flora, and sky. The dimension is survivable atmospheric and pressure wise, but is filled with esoteric creatures of all sizes and shapes and colors, ranging from the size of ants to large buildings. The natural fauna within seem to be interlocked in a constant hyper evolutionary race, making a documentation of species impossible.
```

**Comment by anclandor** [comment]
How does bluespace work for both teleportation and storage?  It seems like the "smaller" size of bluespace would be an issue for bluespace storage.  I mean, how many bluespace bags can you have before people start teleporting into the inside of other people's bags?  What happens when you enter bluespace with a bluespace storage device on your body?

**Comment by tralezab** [comment]
I'm stinky.

**Comment by Cyberboss** [comment]
:(

**Pull request closed without merging**